CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.1.0 (2020-2-8)
```

Comment:
Can you update the date to today? Please

Reply:
Yes that makes totally sense. Adjusted it accordingly